yarn.lock

```diff
@@ -2450,30 +2469,30 @@ __metadata:
   languageName: node
   linkType: hard
 
-"typescript@npm:^5.2.2":
-  version: 5.8.2
-  resolution: "typescript@npm:5.8.2"
+"typescript@npm:5.5.3":
+  version: 5.5.3
```

Comment:
Why are we downgrading TypeScript here (5.8.2 -> 5.5.3)?

Reply:
No idea, i'll fix